templates/CRM/Admin/Page/MailSettings.tpl

```diff
@@ -25,7 +25,7 @@
             <th>{ts}Domain{/ts}</th>
             <th>{ts}Return-Path{/ts}</th>
             <th>{ts}Protocol{/ts}</th>
-            <th>{ts}Source{/ts}</th>
+            <th>{ts}Mail Folder{/ts}</th>
```

Comment:
As above, I think Folder is sufficient here

Reply:
As per my above comment.